accord-core/src/main/java/accord/coordinate/ExecuteTxn.java

```diff
@@ -236,8 +236,9 @@ protected void onDone(Success success, Throwable failure)
                 executeAt = new TimestampWithUniqueHlc(executeAt, uniqueHlc);
             }
 
-            Writes writes = txnId.is(Txn.Kind.Write) ? txn.execute(txnId, executeAt, data) : null;
+            // Always compute Result before Write
```

Comment:
Explain why?

Reply:
It makes it easier for the integration to do things like evaluate part of the result and logic related to what will happen. The order was switched and that caused the integration to apply transactions when the condition failed. I fixed it so that the integration will tolerate either order, but since this is an easy bug to re-introduce and there isn't a huge cost in saying that `Result` and `Write` will be evaluated in a specific order that it would be better to do that.

Reply:
I meant in the code comment for posterity, but thanks I did also want to know 🙂